### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by konwnohing** [realsenseai/realsense-ros#3226] [General]
Why execute this instruction roslaunch realsense2_camera rs_camera.launch   .The following errors will occur [ERROR] [1728614261.666095045]: hwmon command 0x75( 0 1df 0 34f ) failed (response -6= Invalid parameter)
[ WARN] [1728614262.022776230]: Hardware Notification:Left MIPI error,1.72861e+12,Error,Hardware Error

**Comment by MartyG-RealSense** [comment]
Hi @konwnohing  Which librealsense SDK and camera firmware driver versions are you using with the ROS1 wrapper please?  The ideal versions to use with wrapper 2.3.2 are librealsense 2.50.0 or 2.51.1 and firmware 5.13.0.50.
